src/quizzes/chapter-1/1.3.1/unix-quiz.js

```diff
@@ -0,0 +1,47 @@
+const rawQuizdown = `
+---
+shuffleAnswers: true
+---
+
+# If performed on a nonexistent file, all of the following commands will return a non-zero exit code except:
+
+1. [x] touch
+1. [ ] cat
+1. [ ] mv
+1. [ ] cp
+1. [ ] rm
+
+# If you run the following command on a nonexistent directory, what will happen?
+\`\`\`bash
+cd nonexistent-directory-name
+\`\`\`
+
+1. [x] The command will error out and do nothing
+1. [ ] The directory will be created
+1. [ ] The directory will be created AND you will navigate into it
+
+# True or False: This command will yield the same result regardless of what your current working directory is.
+\`\`\`bash
+cd ~
+\`\`\`
+
+1. [x] True
+1. [ ] False
+  > **~** is the shorthand name for the current users home directory, and you can navigate to it from anywhere.
```

Comment:
Let's level this one up a bit

```markdown
Which of the following commands change the directory to the current user's home directory? Check all that apply:

[x] cd /home/$USER
[] cd .
[x] cd ~
[x] cd
```

here is a detail explanation of each command if they get it wrong
>`cd /home/$USER`: This command changes the current directory to the home directory of the user specified by the $USER environment variable.
`cd .`: The . symbol represents the current directory. So, this command essentially means "stay in the current directory."
`cd ~`: This command changes the current directory to the home directory of the current user. The ~ symbol is a shortcut for the user's home directory.
`cd`: Without any arguments, cd defaults to changing the directory to the current user's home directory.

Reply:
I like this a lot better. I'll make the change

Reply:
Added 👍